model_analyzer/config/generate/model_variant_name_manager.py

```diff
@@ -21,95 +22,79 @@ class ModelVariantNameManager:
 
     def __init__(self):
 
-        # Stored as _variant_names[base_model_name][param_combo] = variant_name_string
-        self._variant_names = {}
+        # Dict of {model_config_name: model_config_dict}
+        self._model_config_dicts = {}
```

Comment:
We could also add type hints like `self._model_config_dicts: Dict[str, dict] = {}`.

Reply:
Good catch.